### PR TITLE
h1 header tag added

### PR DESCRIPTION
**Pull request by DestinySTLCC**

### CODE REVIEW

**Comment by Ryan-Greenlee** [comment]
Thank you for the pull request.

**Review by Ryan-Greenlee — COMMENTED** [review]
After this comment, I believe you can mark this as "Resolve Conversation"

**Comment by DestinySTLCC** [comment]
Thank you, Ryan! I'll do that

**Review by Ryan-Greenlee — CHANGES_REQUESTED** [review]
OK to mark conversation as resolved

**Review by Ryan-Greenlee — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -1 +1 @@
-
+<h1>
```
Comment (DestinySTLCC) [inline index.html:1]: Is this the line I need to change?

**Review by Ryan-Greenlee — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (DestinySTLCC) [reply index.html:1]: Thank you! I'll make a quick update.

**Review by Ryan-Greenlee — APPROVED** [review]

**Pull request merged**